README.md

```diff
@@ -65,3 +65,14 @@ In order to run the server:
 ```bash
 python manage.py runserver
 ```
+
+In order to run the tests:
+
+```bash
+pip install pytest-django
```

Comment:
Bunu bence yine requirements'a ekleyelim. Soft-dependancy için ayrı bir sistemimiz olmadığı için hard-dependancy gibi düşünebiliriz

Reply:
Yes doğru ekledim